nuxeo-core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/action/computation/AbstractBulkComputation.java

```diff
@@ -74,18 +76,29 @@ public AbstractBulkComputation(String name, int nbOutputStreams) {
     public void processRecord(ComputationContext context, String inputStreamName, Record record) {
         BulkBucket bucket = BulkCodecs.getBucketCodec().decode(record.getData());
         command = getCommand(bucket.getCommandId());
-        if (command == null) {
-            // this requires a manual intervention, the kv store might have been lost
-            getLog().error(String.format("Stopping processing, unknown command: %s, offset: %s, record: %s.",
-                    bucket.getCommandId(), context.getLastOffset(), record));
-            context.askForTermination();
-            return;
-        }
-        for (List<String> batch : Lists.partition(bucket.getIds(), command.getBatchSize())) {
-            processBatchOfDocuments(batch);
+        if (command != null) {
+            for (List<String> batch : Lists.partition(bucket.getIds(), command.getBatchSize())) {
+                processBatchOfDocuments(batch);
+            }
+            endBucket(context, bucket.getIds().size());
+            context.askForCheckpoint();
+        } else {
+            if (isAbortedCommand(bucket.getCommandId())) {
```

Comment:
as discussed this implementation does not work

Reply:
abort will remove the command from the kv store, so it will work on new computation instance it might requires a node restart but still enable to bypass a blocking command